src/components/MaintenanceBanner/MaintenanceBanner.jsx

```diff
@@ -67,18 +80,17 @@ export class MaintenanceBanner extends Component {
   derivePostContent = () => {
     const { startsAt, expiresAt } = this.props;
 
-    const startsAtET = startsAt.clone().subtract(4, 'hours');
-    const expiresAtET = expiresAt.clone().subtract(4, 'hours');
-
-    if (startsAt.isSame(expiresAt, 'day')) {
+    if (isSameDay(startsAt, expiresAt)) {
       return (
         <>
           <p>
-            <strong>Date:</strong> {startsAtET.format('dddd MMMM D, YYYY')}
+            <strong>Date:</strong>{' '}
+            {formatEastern(startsAt, 'EEEE, MMMM d, yyyy')}
```

Comment:
The `format` tokens used by `date-fns` are different than the custom ones that `moment` uses. [See this doc for more info](https://github.com/date-fns/date-fns/blob/master/docs/unicodeTokens.md).